pkgs/swift2objc/test/integration/global_variables_and_functions_output.swift

```diff
@@ -0,0 +1,56 @@
+// Test preamble text
+
+import Foundation
+
+@objc public class GlobalsWrapper: NSObject {
+  @objc public var globalCustomConstantWrapper: MyOtherClassWrapper {
```

Comment:
Should these be static?

Reply:
Yup, it turns out I didn't add `static` keyword when generating static properties. Glad u caught that I totally missed it for some reason.